### PR TITLE
Add Google Photos Icon

### PR DESCRIPTION
**Pull request by MayaUribe**
## Description
Add Google Photos Icon

## Motivation and Context
For the Google Photos integration in the Publish composer

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style and guidelines of this project. 
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have updated the **CHANGELOG** document.
- [ ] I have added tests to cover my changes.
- [x] I have performed a self-review of my own code
- [ ] I have tested my changes/additions in the latest Chrome, Firefox, and Safari.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] All new and existing tests passed.
- [ ] I have performed the accessibility audit of my UI changes according to the accessibility doc. 
- [ ] [Buffer Engineers] Someone from the Design team reviewed and approved my changes
- [ ] [Buffer Engineers] I have notified the BDS team of my changes in the #proj-design-system Slack channel

### CODE REVIEW

File: src/components/Icon/Icons/Bandcamp.jsx [hunk 1]
```diff
@@ -8,11 +8,18 @@ import createIconComponent from '../utils/createIconComponent';
 const BandcampIcon = createIconComponent({
```
Comment (MayaUribe) [inline src/components/Icon/Icons/Bandcamp.jsx:8]: @gmzjuliana I haven't generated icons in a long time. I executed the `yarn gen:icons` command and that seemed to update a few icons

Comment (MayaUribe) [reply src/components/Icon/Icons/Bandcamp.jsx:8]: It changed the code, but the icons looks the same 😄

**Review by gmzjuliana — DISMISSED** [review]
hey @MayaUribe thanks for the heads up with the icons! I'm not a big fan of pushing the changes for the icons because although they still work, it takes them out of the standard for the icons. Even the GooglePhotos icon follows the same pattern but the revised ones don't

I would suggest two things
1. could you check with the design team if something changed with those icons and if so, if they can ensure that all the icons follow the existing conventions
2. if you're in a rush, I'd consider reverting the file changes for the revised icons and only pushing the changes to googlephotos (and the cachedicondata)

let me know if this sparks any questions!

**Comment by MayaUribe** [comment]
> 2. if you're in a rush, I'd consider reverting the file changes for the revised icons and only pushing the changes to googlephotos (and the cachedicondata)

Thank you @gmzjuliana! I thought about the second option but wasn't sure if it was a good one! thank you! I think for now I'll do that 🤗

**Comment by MayaUribe** [comment]
One question @gmzjuliana, if I reverse the other icons, is it still safe to push the cacheicondata? wouldn't that update something related to the other icons?

**Comment by gmzjuliana** [comment]
i was wondering the same thing @MayaUribe ! jajaj I think it'll be okay though because the file gets re-writen.. Do you mind trying it out though? what happens if you revert the files and try running the gen command again?

Also, totally okay take #2 but do you mind pinging the design team also, so we can get that cleaned up if needed?

**Comment by MayaUribe** [comment]
> i was wondering the same thing @MayaUribe ! jajaj I think it'll be okay though because the file gets re-writen.. Do you mind trying it out though? what happens if you revert the files and try running the gen command again?
> 
> Also, totally okay take option 2 but do you mind pinging the design team also, so we can get that cleaned up if needed?

I just sent a [message](https://buffer.slack.com/archives/C029V2U89/p1661873632559759) to the design team 🙂, and I also tried that out, but it re-generates the icons again 😂

**Comment by MayaUribe** [comment]
@gmzjuliana I'll wait until tomorrow for a reply, otherwise, I'll just reverse the other icons 😄

**Comment by gmzjuliana** [comment]
i think as long as there's no errors, you could merge this with the reverts then @MayaUribe I don't wanna hold you up too much!

**Pull request merged**